camply/notifications/twilio.py

```diff
@@ -0,0 +1,82 @@
+"""
+Push Notifications via Twilio
+"""
+
+import logging
+from typing import List
+
+from camply.config import TwilioConfig
+from camply.containers import AvailableCampsite
+from camply.notifications.base_notifications import BaseNotifications
+
+logger = logging.getLogger(__name__)
+
+
+class TwilioNotifications(BaseNotifications):
+    """
+    Push Notifications via Twilio
+    """
+
+    def __init__(self):
+        try:
+            from twilio.rest import Client
+        except ImportError:
+            raise RuntimeError(
+                "Looks like `twilio` isn't installed. Install it with `pip install camply[twilio]`"
+            )
+
+        if any([TwilioConfig.ACCOUNT_SID == "", TwilioConfig.AUTH_TOKEN == ""]):
+            warning_message = (
+                "Twilio is not configured properly. To send Twilio messages "
+                "make sure to run `camply configure` or set the "
+                "proper environment variable: `TWILIO_ACCOUNT_SID`, `TWILIO_AUTH_TOKEN`."
+            )
+            logger.error(warning_message)
+            raise EnvironmentError(warning_message)
```

Comment:
Can we check for the `ACCOUNT_SID` and `AUTH_TOKEN` to be `None` as well. In the event the environment variables are unset and a `.camply` file doesn't exist these variables would be `None`

Reply:
Good point! Done.